osr2mp4/ImageProcess/Animation/alpha.py

```diff
@@ -39,4 +44,38 @@ def fadein(img, start, end, step):
 		return list_fade(img, start, end, step)
 	else:
 		return img_fade(img, start, end, step)
+		
+		
+""" Gaming Version """
+def img_fade_(img: Image, start: float, end: float, delta: float, duration: float, easing: easings):
+	res = []
+	
+	for time in np.arange(0, duration, delta): # for now
+		fade_val = easing(time, start, end, duration)
+		res += [imageproc.newalpha(img, fade_val)]
+		
+	return res
+	
+def list_fade_(imgs: [Image], start: float, end: float, duration: float, easing: easings):
+	res = []
+	delta = duration/len(imgs)
+	cur_time = 0
+	
+	for img in imgs:
+		fade_val = easing(cur_time, start, end, duration)
+		res += [imageproc.newalpha(img, fade_val)]
```

Comment:
```suggestion
		res.append(imageproc.newalpha(img, fade_val))
```

Reply:
yea so about this, idk why i started doing the `list += [val]` thing. somehow it just works in my brain